mongo/integration/unified_spec_test.go

```diff
@@ -283,6 +284,24 @@ func runSpecTestCase(mt *mtest.T, test *testCase, testFile testFile) {
 
 		// Reset the client using the client options specified in the test.
 		testClientOpts := createClientOptions(mt, test.ClientOptions)
+
+		// If the CRYPT_SHARED_LIB_PATH environment variable is set, AutoEncryptionOptions is set, and
+		// AutoEncryption isn't disabled (neither bypassAutoEncryption nor bypassQueryAnalysis are
+		// true), then add extra options to load and require the crypt_shared library.
+		if path := os.Getenv("CRYPT_SHARED_LIB_PATH"); path != "" && testClientOpts.AutoEncryptionOptions != nil {
```

Comment:
Can you use `getCryptSharedLibExtraOptions` here somehow? Or is it difficult to avoid an import cycle?

Reply:
That's a good idea. They're in the same package, so no import cycle issues, just not something I thought of. Updated to use `getCryptSharedLibExtraOptions` here.

Reply:
[sa-lint](https://spruce.mongodb.com/task/mongo_go_driver_static_analysis_sa_lint_patch_6f97e181d6c20f9be2e81be536d83fd47c99484d_62a96d27a4cf4779a9ab9bce_22_06_15_05_24_56/logs?execution=0) has this error:

```
[2022/06/15 05:49:36.435] GOOS=linux GOARCH=amd64 golangci-lint run --config .golangci.yml ./...
[2022/06/15 05:49:36.435] mongo/integration/unified_spec_test.go:300:23: undeclared name: `getCryptSharedLibExtraOptions` (typecheck)
[2022/06/15 05:49:36.435] 				for k, v := range getCryptSharedLibExtraOptions() {
```

I am not sure why.